### ISSUE TITLE
How to evaluate the transformer on the summarization task by using the tensor2tensor?

### ISSUE THREAD

**Issue by zhaoguangxiang** [tensorflow/tensor2tensor#1310]
### Description
e.g. CNN/dailymail dataset. I know how to train the transformer on the dataset, but  I don't know how to evaluate the performance of the transformer on the task.

**Comment by pig60214** [comment]
hello, Do you get the answer?

**Comment by zhaoguangxiang** [comment]
No, I didn' t, and I notice that there is a get_rouge.py in the utils directory

**Comment by nxphi47** [comment]
Hi, have anyone got the answer yet? I also have question about this.

**Comment by zhaoguangxiang** [comment]
> Hi, have anyone got the answer yet? I also have question about this.

not yet

**Comment by RyanOngAI** [comment]
Hope its not too late. Run t2t-decoder, which would generate three output files, two of which is summary generated by your model and the other its the gold-summary. Install pyrouge and feed those two output and you will get R-1, R-2 and R-L
